source/content/guides/new-dashboard/02-workspaces.md

```diff
@@ -62,6 +62,36 @@ Manage team members and assign [roles and permissions](/change-management#roles-
 
 Once it's enabled, use it to visually compare changes to the site. Visit the [Autopilot documentation](/guides/autopilot) for more information.
 
+### Webhooks
+
+Webhooks allows you to set up integrations from third-party applications and subscribe to different events in the [New Dashboard](/guides/new-dashboard). When an event is triggered, you will receive a notification. This feature is workspace-specific, so anyone who has Admin permissions to a particular workspace has access.
+
+#### Create A Webhook
+
+1. Navigate to the workspace where you want to set up a webhook > Click <span class="glyphicons glyphicons-cogwheel"></span> **Settings** > Click **Webhooks**.
+
+1. Click **+ Create New Webhook**.
+
+1. Name your Webhook.
```

Comment:
May want to note that this name is only used as an identifier within the dashboard.